db/schema.rb

```diff
@@ -10,18 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161130153206) do
```

Comment:
Yeah it is weird one. it kept re-occuring. I just checked it with the intention to remove it later on.